src/odemis/driver/test/pmtctrl_test.py

```diff
@@ -261,6 +267,54 @@ def receive_image(self, dataflow, image):
         self.assertEqual(self.pmt.data.active, False)
         self.is_received.set()
 
+    def test_while_acquiring(self):
+        """
+        While acquiring the underline protection is always off
+        """
+        self.acq_left = 10
+        self.pmt_protection_recorded = []
+        self.pmt_protection_expected = [False] * self.acq_left
+        self.acq_complete = threading.Event()
+
+        # Protection should be on before start acquisition
+        self.assertEqual(self.control.protection.value, True)
+        self.assertEqual(self.pmt.data.active, False)
+
+        logging.debug("Acquisition starts..")
+        self.pmt.data.subscribe(self.acquire_data)
+
+        # Protection should be off while acquiring
+        self.assertEqual(self.control.protection.value, False)
+        # wait the last point is fully acquired
+        self.acq_complete.wait()
+        self.assertEqual(self.acq_left, 0)
+        # the protection remains off during the acquisition
+        self.assertEqual(self.pmt_protection_recorded, self.pmt_protection_expected)
+
+        # the protection remains off after settle time (10e-6) because delay is 0.1
+        time.sleep(self.scanner.settleTime)
```

Comment:
Well, settle time and protection delay have no links, so let's drop this.
If you want to say "wait a tiny bit", then just write `time.sleep(1e-3)`, to make it clear it's just an arbitrary small number